Dockerfile

```diff
@@ -1,6 +1,5 @@
 FROM registry.svc.ci.openshift.org/ocp/builder:golang-1.13 AS builder
 WORKDIR /go/src/github.com/openshift/openshift-controller-manager
-RUN yum install -y gpgme-devel libassuan-devel
```

Comment:
Any special reason why was this here? I mean, I am having problems [building release 4.4](https://github.com/openshift/openshift-controller-manager/pull/88) of this repository, most likely due to these dependencies. I may go ahead and remove them as well but I would like to check with you first.

Reply:
Goes back to the origins of this stuff (even predates back to when all of this was in origin).

As time has passed, these installs were done in the base image, and as I was told the clean up here had simply been overlooked.

From my investigations `registry.svc.ci.openshift.org/ocp/builder:golang-1.12` had these RPMs as well.  The yum install simply was less flaky as a result of an imagebuilder diff between the 1.12 and 1.13 images.

Bottom line, I would +1 @ricardomaraschini removing that line in the 4.4 dockerfiles.